sdk/cosmos/cosmos-emulator-internal-matrix.json

```
@@ -0,0 +1,112 @@
+{
+    "displayNames": {
+      "--disablecov": "",
+      "false": "",
+      "true": ""
+    },
+    "matrix": {
+      "Agent": {
+        "windows-2022": { "OSVmImage": "env:WINDOWSVMIMAGE", "Pool": "env:WINDOWSPOOL" }
+      },
```

This would at least let you de-dup this part without any title change, I think

```suggestion
      },
      "Account": {
        "_": {
          "ACCOUNT_HOST": "https://localhost:8081/",
          "ACCOUNT_KEY": "C2y6yDjf5/R+ob0N8A7Cgv30VRDJIWEHLM+4QDU5DE2nQ9nDuVTqobD4b8mGGyPMbIZnqyMsEcaGQy67XIw/Jw==",
        }
      },
```

---

Going to try this.